druid-lookups/src/main/java/com/yahoo/maha/maha_druid_lookups/query/lookup/RocksDBLookupExtractor.java

```diff
@@ -63,8 +63,11 @@ public boolean canIterate() {
             RocksIterator it = db.newIterator();
             it.seekToFirst();
             while (it.isValid()) {
-                byte[] cacheByteValue = db.get(it.key());
                 Parser<Message> parser = schemaFactory.getProtobufParser(extractionNamespace.getNamespace());
+                byte[] cacheByteValue = db.get(it.key());
+                if (cacheByteValue == null) {
+                    return tempMap.entrySet();
```

Comment:
if one value is null, it should continue looking for next values.

Reply:
Fixed